.github/workflows/lint.yml

```diff
@@ -4,15 +4,15 @@ on: pull_request
 
 jobs:
   lint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
This is probably not desirable - the Dockerfile should match the version, and I don't think github and docker repos' definition of latest is same. (Not repeating it over later points.)

Reply:
Right, didn't think about the Dockerfile. Bumping both to 22.04!

Reply:
Oddly enough, `latest` means `20.04` still (https://github.com/actions/runner-images)